Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.5
+
+MAINTAINER eric.kascic@stelligent.com
+
+RUN gem install cfn-nag
```

Comment:
i'd suggest we pin this based upon the version the pipeline pushes to rubygems.... and then the label of the image can reflect that version

Reply:
I was going to do this when we figured out DockerHub and start publishing. This was just a starting point.